pool/database/migration/migration_schedule.ml

```diff
@@ -0,0 +1,22 @@
+let create_schedule_table =
+  Sihl.Database.Migration.create_step
+    ~label:"create registered schedule table"
+    {sql|
+      CREATE TABLE IF NOT EXISTS pool_schedules (
```

Comment:
Shouldn't there be a reference to the tenant? (`database_label`)

Reply:
Maybe in a later version. Currently there are only cross tenant processes.